jenkins-integration/scripts/foss_install.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+
+export GENCONFIG_LAYOUT="${GENCONFIG_LAYOUT:-centos6-64ma}"
+export BEAKER_TESTSUITE="${BEAKER_TESTSUITE:-beaker/install/foss/}"
+export BEAKER_KEYFILE="~/.ssh/id_rsa-acceptance"
+export BEAKER_HELPER="beaker/helper.rb"
+export BEAKER_INSTALL_TYPE="foss"
```

Comment:
This Beaker `--type` argument determines file paths for a bunch of things, which in the past have been different between FOSS and PE installations.  In the new world, the paths for FOSS and PE have been unified under new locations, which are available with the `aio` argument.

I think this means we can always use `--type aio` for these scripts, regardless of FOSS or PE.  This would only break things if we try running against pre-AIO stuff, but I'm not sure that we're going to.

@jpinsonault If we run this with `"aio"` and it works, then I think we should use that.  I'll test this out in a minute.